snykTask/src/index.ts

```diff
@@ -146,21 +153,39 @@ async function runSnykTest(
 
   const snykTestToolRunner = tl
     .tool(snykPath)
+    .argIf(taskArgs.testType == TestType.CODE, 'code')
+    .argIf(
+      taskArgs.dockerImageName || taskArgs.testType == TestType.CONTAINER_IMAGE,
+      'container',
+    )
     .arg('test')
     .argIf(
-      taskArgs.severityThreshold,
+      taskArgs.testType != TestType.CODE && taskArgs.severityThreshold,
```

Comment:
question: why did you decide to differentiate threshold by product?

Reply:
because snyk code severity thresholds are only low, medium, high. so a new argument field `codeSeverityThreshold` is introduced at the pipeline editor so the combo box will only show these 3.

Reply:
Let's go with just one severity setting and map critical to high.

Reply:
this is misleading to the user, and our products have different thresholds.  Selecting critical would have no effect, why should it be presented to the user as an option?

Reply:
I don't agree, but this is not a blocker.